CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/ImagingHelperTests.cs

```diff
@@ -55,9 +56,23 @@ public async Task ImagingTest()
 			// Act
 			// Assert
 			Assert.DoesNotThrow(() => Sut.ImageAllDocumentsInWorkspaceAsync(workspaceArtifactId).Wait());
+		  Assert.IsTrue(Sut.CheckThatAllDocumentsInWorkspaceAreImaged(workspaceArtifactId).Result);
 
 			//Cleanup
 			await WorkspaceHelper.DeleteSingleWorkspaceAsync(workspaceArtifactId);
 		}
+
+		private void CleanupWorkspaceIfItExists(string workspaceName)
+		{
+			try
+			{
+				int existingWorkspaceArtifactId = WorkspaceHelper.GetFirstWorkspaceArtifactIdQueryAsync(workspaceName).Result;

```

Comment:
It's possible there could be multiple workspaces with the same name. Can we update it to delete all workspaces with the same name.